sdk/ml/azure-ai-ml/tests/test_configs/automl_job/test_datasets/image_instance_segmentation/train/train_annotations.jsonl

```diff
@@ -0,0 +1,102 @@
+{"image_url": "azureml://subscriptions/102a16c3-37d3-48a8-9237-4c9b1e8e80e0/resourcegroups/risha-rg/workspaces/risha-ws-centraluseuap/datastores/workspaceblobstore/paths/LocalUpload/57e0ce1a06d180c7be40a8dc137a09c5/odFridgeObjectsMask/images/104.jpg", "image_details": {"format": "jpg", "width": 499, "height": 666}, "label": [{"label": "can", "bbox": "null", "isCrowd": 0, "polygon": [[0.5951903807615231, 0.6141141141141141, 0.5711422845691383, 0.6141141141141141, 0.5511022044088176, 0.6081081081081081, 0.43286573146292584, 0.53003003003003, 0.3807615230460922, 0.4984984984984985, 0.3687374749498998, 0.48348348348348347, 0.3667334669338677, 0.46846846846846846, 0.3587174348697395, 0.4594594594594595, 0.35671342685370744, 0.44894894894894893, 0.3587174348697395, 0.4294294294294294, 0.37074148296593185, 0.41291291291291293, 0.4088176352705411, 0.3888888888888889, 0.42685370741482964, 0.3858858858858859, 0.4709418837675351, 0.3858858858858859, 0.48697394789579157, 0.39039039039039036, 0.5290581162324649, 0.4099099099099099, 0.5511022044088176, 0.4174174174174174, 0.6172344689378757, 0.44894894894894893, 0.6492985971943888, 0.46096096096096095, 0.655310621242485, 0.46546546546546547, 0.6693386773547094, 0.46846846846846846, 0.6893787575150301, 0.4804804804804805, 0.6973947895791583, 0.4894894894894895, 0.7034068136272545, 0.506006006006006, 0.7034068136272545, 0.5285285285285285, 0.6973947895791583, 0.5465465465465466, 0.6733466933867736, 0.5795795795795796, 0.6432865731462926, 0.5990990990990991, 0.6192384769539078, 0.6081081081081081, 0.5971943887775552, 0.6111111111111112, 0.5951903807615231, 0.6141141141141141]]}]}
```

Comment:
maybe also remove the simplification from dependencies - https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/ml/azure-ai-ml/dev_requirements.txt#:~:text=%23%20comment%20this%20line,platform.python_implementation!%3D%22PyPy%22